crates/context/src/cfg.rs

```diff
@@ -68,6 +68,9 @@ pub struct CfgEnv<SPEC = SpecId> {
     /// Skips the checks related to eip 7702 and enables it.
     #[cfg(feature = "enable_eip7702")]
     pub enable_eip7702: bool,
+    /// Skips the checks related to eip 7623 and enables it.
```

Comment:
What does "skips the checks" mean?

Reply:
I'll update the doc comment, would make more sense to have "enables EIP-7623, regarless of the current fork".